include/pfcp_packet.hrl

```diff
@@ -1488,3 +1488,7 @@
 -record(tp_created_nat_binding, {
 	  group
 }).
+
+-record(tp_ipfix_policy, {
+	  policy = <<>>
```

Comment:
I'd suggest removing the default value to make the user explicit about its use.

Reply:
But why would we generate a default? Just asking.

Reply:
(I'm fairly anti-default in any case, but I have no wish to impose this view).

Reply:
The default is there so that for complex IEs you don't need to initialize all the fields all the time.
For example, most flags are initialized to either 0 or [] (depending on their type).

I would be possible to come up with a more complex syntax to include defaults only for some types, but so far there has be no need to that.